CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Tasks
 * Build: Visueel regressietesten ([#2588](https://github.com/dso-toolkit/dso-toolkit/issues/2588))
+* Packages: Dependencies updates ([#2675](https://github.com/dso-toolkit/dso-toolkit/issues/2675))
+* Packages: angular updaten naar v18 ([#2676](https://github.com/dso-toolkit/dso-toolkit/issues/2676))
```

Comment:
Ik mis het nut van deze vermelding, en eigenlijk het nut van de issue: Het lijkt erop alsof de update pijnloos is gegaan? Kan deze daarom niet gewoon mee in het reguliere update issue?

Overigens staat er in het Angular update issue wel nog een interessante opmerking:

> Tevens maximaal 3 Angular versies ondersteunen in de distributable package.json. Dat betekent dat ^15 weggaat en dat dit een breaking release is.

Heb je hier nog naar gekeken?

Reply:
Omdat dit een major release van angular betrof heb ik de angular update losgetrokken van de reguliere packages update. Maar achteraf bleek het niet veel voeten in de aarde te hebben. Dus heb ik de angular update in deze PR meegenomen. Maar omdat vanwege de, nu inmiddels wel bijgewerkte peerDependencies van `@angular/common`, `@angular/core` en `@angular/cdk` naar `^16.0.0 || ^17.0.0 || ^18.0.0` en dat dat een breaking release maakt heb ik de angular update wel vermeld in de `CHANGELOG.md`